chains/solana/contracts/programs/ccip-router/src/instructions/v1/admin.rs

```diff
@@ -0,0 +1,361 @@
+use anchor_lang::prelude::*;
+use anchor_spl::token_interface;
+
+use crate::{
+    AcceptOwnership, AddBillingTokenConfig, AddChainSelector, BillingTokenConfig, CcipRouterError,
```

Comment:
Is there any cyclic dependency between files?

Reply:
Mostly no. There is only one cycle right now, which is easy to break if you prefer.

The `lib.rs` is the only non-versioned file that imports versioned code. But the versioned code also imports the errors and constants that are listed in the `lib.rs`. This is not a problem in Rust, as I've tested this code locally and it's passing CI, but then again it's trivial for me to move errors and constants to separate files if you'd prefer.